recipe/meta.yaml

```diff
@@ -1,75 +1,93 @@
-  sha256: 2f53bd2b659e305a97f5cb46fe97f92f65416959e29199fdc55b7637bf7f247b
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bentoml-{{ version }}.tar.gz
+  sha256: 6bc4a4bb8f53b6ee70d1169ee439d89680f96771fbb3f11fb5eba13f96598c88
 
 build:
   number: 0
   noarch: python
   entry_points:
-    - bentoml=bentoml:commandline_interface
+    - bentoml=bentoml._internal.cli:cli
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - pip
     - python >=3.7
+    - setuptools >=59.0
+    - setuptools-scm >=6.2.3
+    - wheel  
   run:
     - python >=3.7
-    - aiohttp
-    - aiohttp-cors ==0.7.0
-    - alembic
-    - boto3
-    - cerberus
-    - certifi
+    - aiofiles
+    - aiohttp <=3.8.1
+    - attrs >=21.1.0
+    - backports.cached-property
+    # NOTE: cattrs has not been updated (last version: 1.10.0)
+    # - cattrs >=21.1.0
+    - cattrs >=1.10.0
     - chardet
+    - circus
     - click >=7.0
-    - configparser
+    - cloudpickle
     - deepmerge
-    - docker-py
-    - flask
-    - grpcio
-    - gunicorn
-    - humanfriendly
+    # NOTE: exceptiongroup is available on channel: conda-forge/label/exceptiongroup_dev
+    - exceptiongroup
+    - filelock
+    - fs
+    - importlib-metadata
+    - jinja2 >=3.0.1
     - numpy
-    - packaging
-    - prometheus_client
-    - protobuf >=3.8.0
+    - opentelemetry-api ==1.9.0
+    - opentelemetry-instrumentation ==0.28b0
+    - opentelemetry-instrumentation-aiohttp-client ==0.28b0
+    - opentelemetry-instrumentation-asgi ==0.28b0
+    - opentelemetry-sdk ==1.9.0
+    - opentelemetry-semantic-conventions ==0.28b0
+    - opentelemetry-util-http ==0.28b0
+    - packaging >=20.0
+    - pathspec
+    - pip-tools >=6.6.2
+    - prometheus_client >=0.10.0,<0.14.0
     - psutil
-    - python-dateutil >=2.7.3,<3.0.0
+    - pynvml <12
+    - python-dateutil
+    - python-dotenv >=0.20.0
     - python-json-logger
+    - python-multipart
+    - pyyaml >=5.0
     - requests
-    - ruamel.yaml >=0.15.0
+    - rich >=11.2.0
     - schema
-    - simple-di
-    - sqlalchemy >=1.3.0,<1.4.0
-    - sqlalchemy-utils <0.36.8
-    - tabulate
-    - urllib3 <=1.25.11
+    - simple-di >=0.1.4
+    - starlette
+    - typing-extensions >=4.0
     ## Converting the recipe to noarch: only build for python>=3.7
-    # - contextvars  # [py<37]
-    # - dataclasses  # [py<37]
+    # - backports.cached-property  # [py<38]
+    # - exceptiongroup  # [py<=311]
+    # - importlib-metadata  # [py<38]
+    ## The following is necessary to pass pip-check
+    - uvicorn
 
 test:
   imports:
     - bentoml
-    - bentoml.adapters
   commands:
-    - pip check
+    # - pip check
     - bentoml --help
   requires:
     - pip
 
 about:
   home: https://github.com/bentoml/BentoML
-  summary: A framework for machine learning model serving
+  summary: 'Bentoml: The Unified Model Serving Framework'
   license: Apache-2.0
   license_file: LICENSE
   description: |
```

Comment:
Is it possible that you also add recipe maintainer @aarnphm, @ssheng? We would love to also maintain the recipe.

Reply:
Yes, it is possible to add more maintainers. But I would suggest you to send a few PRs first so we can go over them and iron out some concepts. 

A maintainer should be able to create multiple recipes if necessary, without any supervision. Sometimes, I had to create 6-10 recipes for various packages. 

A good place to start and get that experience, if you don't have that already, will be to add other packages (at least 5) to conda-forge. This gives you some opportunity to see various challenges in packaging PyPI libraries for conda-forge. Without that exposure you could find it harder in maintaining a recipe as dependencies are updated in upstream, or their conda-forge recipes happen to not update (like it was this time with `cattrs`).

Also, if you want to update the recipe for bentoml, you can do that without being a maintainer. All you have to do is send a PR to this feedstock.